isis/src/base/apps/nocam2map/main.cpp

```diff
@@ -48,738 +48,30 @@ map <QString, void *> GuiHelpers() {
-}
-
-
-// Function to delete unwanted tables in header
-void DeleteTables(Pvl *label, PvlGroup kernels) {
-  //Delete any tables in header corresponding to the Kernel
-  const QString tableStr("Table");
-  const QString nameStr("Name");
-
-
-  //Setup a list of tables to delete with predetermined values and any tables in the kernel.
-  //If additional tables need to be removed, they can be added to the list of tables that
-  //detine the 'tmpTablesToDelete' QString array directly below.  
-  QString tmpTablesToDelete[] = {"SunPosition","BodyRotation","InstrumentPointing",
-                                                              "InstrumentPosition"};
-  std::vector<QString> tablesToDelete;
-  int sizeOfTablesToDelete = (int) sizeof(tmpTablesToDelete)/sizeof(*tmpTablesToDelete);
-  for (int i = 0; i < sizeOfTablesToDelete; i++) {
-    tablesToDelete.push_back( tmpTablesToDelete[i] );
+    // in this case, output data are in a "Results" group.
+  PvlGroup results = appLog.findGroup("Mapping");
+  if( ui.WasEntered("TO") && ui.IsInteractive() ) {
+    Application::GuiLog(results);
   }
-  for (int j=0; j < kernels.keywords(); j++) {   
-    if (kernels[j].operator[](0) == tableStr)  {
-      bool newTableToDelete=true;
-      for (int k = 0; k<sizeOfTablesToDelete; k++) {
-        if ( tablesToDelete[k] == kernels[j].name() ) {
-          newTableToDelete=false;
-          break;
-        }
-      }
-      if (newTableToDelete) {
-        tablesToDelete.push_back( kernels[j].name() );
-        sizeOfTablesToDelete++;
-      }
-    }
-  } 
-  int tablesToDeleteSize = (int) tablesToDelete.size();
-  //Now go through and find all entries in the label corresponding to our unwanted keywords
-  std::vector<int> indecesToDelete;
-  int indecesToDeleteSize=0;
-  for (int k=0; k < label->objects(); k++) {
-    PvlObject &currentObject=(*label).object(k);
-    if (currentObject.name() == tableStr) {
-      PvlKeyword &nameKeyword = currentObject.findKeyword(nameStr);
-      for (int l=0; l < tablesToDeleteSize; l++) {
-        if ( nameKeyword[0] == tablesToDelete[l] ) {
-          indecesToDelete.push_back(k-indecesToDeleteSize);
-          indecesToDeleteSize++;
-          //(*label).deleteObject(k);
-          //tableDeleted = true;
-          break;
-        }
-      }
-    }
-  }
-  //Now go through and delete the corresponding tables
-  for (int k=0; k < indecesToDeleteSize; k++) {
-    (*label).deleteObject(indecesToDelete[k]);
-  }
-}
-
-
-int nocam2map::OutputSamples() const {
-  return p_outputSamples;
 }
 
-int nocam2map::OutputLines() const {
-  return p_outputLines;
-}
-
-// Helper function to print out mapfile to session log
-void PrintMap() {
-  UserInterface &ui = Application::GetUserInterface();
-
-  // Get mapping group from map file
-  Pvl userMap;
-  userMap.read(ui.GetFileName("MAP"));
-  PvlGroup &userGrp = userMap.findGroup("Mapping", Pvl::Traverse);
-
-  //Write map file out to the log
-  Isis::Application::GuiLog(userGrp);
-}
```

Comment:
Any particular reason this got moved?

Reply:
Gui Helpers got shuffled when moved back into Main.cpp but it's still there. https://github.com/USGS-Astrogeology/ISIS3/blob/ef3f93470d3554c717c8b266fab07320a281e649/isis/src/base/apps/nocam2map/main.cpp#L118